extensions/kotlin/deployment/src/main/java/io/quarkus/kotlin/deployment/KotlinCompilationProvider.java

```diff
@@ -43,7 +43,7 @@ public void compile(Set<File> filesToCompile, Context context) {
         }
 
         if (messageCollector.hasErrors()) {
-            throw new RuntimeException("Compilation failed" + String.join("\n", messageCollector.getErrors()));
+            throw new RuntimeException("Compilation failed" + String.join("%n", messageCollector.getErrors()));
```

Comment:
@rsvoboda I'm not sure this one is correct so I thought I might as well ask. `%n` is supported by `String.format()` but is this string passed to it?

Reply:
change reverted, thanks!